airflow/providers/elasticsearch/log/es_task_handler.py

```diff
@@ -321,7 +321,7 @@ def emit(self, record):
             setattr(record, self.offset_field, int(time() * (10**9)))
             self.handler.emit(record)
 
-    def set_context(self, ti: TaskInstance) -> None:
+    def set_context(self, ti):
```

Comment:
why do you need to remove the type hints?

Reply:
The static checks were failing wrt to superclass annotations. Essentially the return values are different between the subclass and superclass. The superclass returns a non-None value whereas this returns None. If I specify partly the annotation e.g. ti: TaskInstance, it still complains that it does not match the superclass signature.

Reply:
Added the type annotation back with   `# type: ignore[override]`

Reply:
Why not match the superclass definition?

Reply:
okay, removed type override ignore by accepting kwargs. Hope that's okay if tests pass.